js/script.js

```diff
@@ -93,17 +93,17 @@ function getRandomQuote () {
   return quotes[getRandomQuote];
 }
 console.log(getRandomQuote());
-
 /***
  * `printQuote` function
 ***/
 function printQuote () {
```

Comment:
semantic naming of your variables and functions is very important for you  and others to understand what your code is doing...so think about the naming carefully.  In this case i think 'displayQuote' or 'renderQuote' is better..b/c you aren't actually printing anything